test/hotspot/jtreg/compiler/c2/TestMergeStores.java

```diff
@@ -797,11 +801,13 @@ static Object[] test2RBE(byte[] a, int offset, long v) {
     }
 
     @Test
-    @IR(counts = {IRNode.STORE_B_OF_CLASS, "byte\\\\[int:>=0] \\\\(java/lang/Cloneable,java/io/Serializable\\\\)", "8",
-                  IRNode.STORE_C_OF_CLASS, "byte\\\\[int:>=0] \\\\(java/lang/Cloneable,java/io/Serializable\\\\)", "0",
-                  IRNode.STORE_I_OF_CLASS, "byte\\\\[int:>=0] \\\\(java/lang/Cloneable,java/io/Serializable\\\\)", "0",
-                  IRNode.STORE_L_OF_CLASS, "byte\\\\[int:>=0] \\\\(java/lang/Cloneable,java/io/Serializable\\\\)", "0"},
-        applyIfPlatform = {"little-endian", "true"})
+    @IR(counts = { IRNode.STORE_B_OF_CLASS, "byte\\\\[int:>=0] \\\\(java/lang/Cloneable,java/io/Serializable\\\\)", "1",
+                   IRNode.STORE_C_OF_CLASS, "byte\\\\[int:>=0] \\\\(java/lang/Cloneable,java/io/Serializable\\\\)", "0",
+                   IRNode.STORE_I_OF_CLASS, "byte\\\\[int:>=0] \\\\(java/lang/Cloneable,java/io/Serializable\\\\)", "0",
+                   IRNode.STORE_L_OF_CLASS, "byte\\\\[int:>=0] \\\\(java/lang/Cloneable,java/io/Serializable\\\\)", "1",
+                   IRNode.REVERSE_BYTES_L, "1"},
+        applyIf = {"UseUnalignedAccesses", "true"},
+        applyIfPlatformAnd = {"little-endian", "true", "riscv64", "false"})   // Exclude RISCV64 because ReverseBytes are not supported
```

Comment:
Seems to me the code comment could be made more accurate. In fact, all the `ReverseBytes` variants will be available on riscv64 if we have the Zbb extension [1]. And I see the IR matching added here in this test works on such platforms. I wonder if there is an easy way to enable these IR tests for these platforms.

[1] https://github.com/openjdk/jdk/blob/master/src/hotspot/cpu/riscv/riscv_b.ad#L181

Reply:
@RealFYang , thanks for your comments. As you mentioned, IR test framework can not check Zbb extension for RISCV platform, so I disable it for RISCV64. I think we can make a new PR to enhance IR test framework and enable the test.

Reply:
That OK to me. Maybe you can update the code comments to make it more accurate? Like:
`// Exclude riscv64 where ReverseBytes is only conditionally supported`

Reply:
Comments updated

Reply:
Thank you!